packages/create-plugin/src/commands/generate.plopfile.ts

```diff
@@ -113,6 +113,22 @@ export default function (plop: NodePlopAPI) {
         ? getActionsForTemplateFolder({ folderPath: backendFolderPath, exportPath })
         : [];
 
+      // Common, pluginType and backend actions can contain the same destination filenames.
+      // This filtering removes the duplicates to prevent plop erroring and makes sure the correct
+      // template is scaffolded.
```

Comment:
Maybe add something to the comment like "last actions take precedence" - to explain the `.reduceRight(...).reverse()`?

Reply:
Decided to take a different approach and order the array spread items backwards so we `reduce` rather than `reduceRight(...).reverse()`. Updated the comment as suggested to explain precedence.